assets/js/Station.jsx

```diff
@@ -56,34 +82,98 @@ function Station({
       </h3>
       <div className="row">
         <div className="col">
-          {makeSign(config, 's', signs, currentTime, line, signConfigs, setConfigs, readOnly)}
-          {makeSign(config, 'w', signs, currentTime, line, signConfigs, setConfigs, readOnly)}
+          {makeSign(
+            config,
+            's',
+            signs,
+            currentTime,
+            line,
+            signConfigs,
+            setConfigs,
+            readOnly,
+          )}
+          {makeSign(
+            config,
+            'w',
+            signs,
+            currentTime,
+            line,
+            signConfigs,
+            setConfigs,
+            readOnly,
+          )}
         </div>
         <div className="col">
-          {makeSign(config, 'c', signs, currentTime, line, signConfigs, setConfigs, readOnly)}
-          {makeSign(config, 'm', signs, currentTime, line, signConfigs, setConfigs, readOnly)}
+          {makeSign(
+            config,
+            'c',
+            signs,
+            currentTime,
+            line,
+            signConfigs,
+            setConfigs,
+            readOnly,
+          )}
+          {makeSign(
+            config,
+            'm',
+            signs,
+            currentTime,
+            line,
+            signConfigs,
+            setConfigs,
+            readOnly,
+          )}
         </div>
         <div className="col">
-          {makeSign(config, 'n', signs, currentTime, line, signConfigs, setConfigs, readOnly)}
-          {makeSign(config, 'e', signs, currentTime, line, signConfigs, setConfigs, readOnly)}
+          {makeSign(
+            config,
+            'n',
+            signs,
+            currentTime,
+            line,
+            signConfigs,
+            setConfigs,
+            readOnly,
+          )}
+          {makeSign(
+            config,
+            'e',
+            signs,
+            currentTime,
+            line,
+            signConfigs,
+            setConfigs,
+            readOnly,
+          )}
         </div>
       </div>
       <hr />
     </div>
   );
 }
 
+const zoneConfig = PropTypes.shape({
+  value: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
+  modes: PropTypes.shape({
+    auto: PropTypes.bool,
+    custom: PropTypes.bool,
+    headway: PropTypes.bool,
+    off: PropTypes.config,
```

Comment:
Why is this `PropTypes.config` instead of `bool` like the others?

Reply:
hmm that's weird. just a typo.